db/migrate/20161212205412_add_display_name_to_tutorials.rb

```diff
@@ -0,0 +1,9 @@
+class AddDisplayNameToTutorials < ActiveRecord::Migration
+  def change
+    add_column :tutorials, :display_name, :text, default: ""
+    Tutorial.find_each do |tut|
```

Comment:
I was guessing there was a method like that but I couldn't track it down in the docs. #find_each was used in other migrations so I went with it. Updating now.